knative-operator/deploy/console_cli_download.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: console.openshift.io/v1
```

Comment:
Let's move this to `deploy/resources/` which is the (arbitrary) conventional place for any yaml applied by the operator. Humans are expected to apply the stuff in `deploy/`